src/test/regress/sql/merge_arbitrary.sql

```diff
@@ -131,3 +131,48 @@ BEGIN;
 EXECUTE local(0, 1);
 SELECT * FROM t1 order by id;
 ROLLBACK;
+
+-- Test prepared statements with repartition
+PREPARE merge_repartition_pg(int,int,int,int) as
```

Comment:
The point is that the same tables such as `citus_target/citus_source` are already going to be tested without being distributed tables in arbitrary configs, so it would be a redundant and a little confusing. We never use special local tables in arbitrary configs for that reason.

See https://github.com/citusdata/citus/blob/main/src/test/regress/sql/postgres.sql which does the trick in `PostgresConfig` under src/test/regress/citus_tests/config.py

Reply:
Yes, pg tables were causing lot of issues in arbitrary, I changed them to Citus-local and tests are clean now.